app/models/immunisation_import.rb

```diff
@@ -116,9 +134,18 @@ def initialize(data:, team:)
     def to_location
       return unless valid?
 
-      Location.new(
-        name: @data["SCHOOL_NAME"].strip,
-        urn: @data["SCHOOL_URN"].strip
+      Location.new(name: school_name, urn: school_urn)
+    end
+
+    def to_patient
+      return unless valid?
+
+      Patient.new(
```

Comment:
Do we want to do a "update or create" here?

Currently this won't work if we import the same patient twice across two different files as the NHS number unique constraint will fail.

Reply:
Added a `.reject(&:invalid?)` instead which is consistent to how we handle Locations

We could instead branch here and return `nil` if the record exists and `.compact` instead back in the `process!` method, I think either approach is fine

Reply:
Does this introduce a risk that we'll ignore these records for a different validation error (to uniqueness), and that'll then get swallowed and we won't know why the import fails?